examples/ingest/cmd/root.go

```diff
@@ -110,6 +112,19 @@ func initConfig() {
 	if err != nil {
 		er(fmt.Sprintf("failed to create IndyKite Ingest Client: %v", err))
 	}
+	retryClient, err = ingest.NewRetryClient(context.Background(),
+		&ingest.RetryPolicy{
+			MaxAttempts:       4,
+			InitialBackoff:    "1s",
+			BackoffMultiplier: 2,
+			CtxTimeout:        120 * time.Second,
+		},
+		grpc.WithCredentialsLoader(apicfg.DefaultEnvironmentLoader),
+		grpc.WithRetryOptions(retry.Disable()),
```

Comment:
Maybe we should hardcode the `grpc.WithRetryOptions(retry.Disable())` in the retry client? Looks a bit weird to pass `retry.Disable()` to a retry client 😅

Reply:
fixed

Reply:
Good question, and the answer is "kinda". The built-in gRPC retry functionality does not work for streams, so if you call the stream function of a client with the retry options enabled, you will get a gRPC error. But it would be useful for the single ingest function (non-stream), which the client also has.

Reply:
You cannot allow retry since the client contains stream methods.
![image](https://github.com/indykite/indykite-sdk-go/assets/5268084/2ff19c70-a15b-4654-8195-d88b5c2cda26)

Reply:
Yup, so we would need to find a way to enable that only in the single `IngestRecord()` function, but not in the stream. Not important for now, and out of scope for this PR